sys/sys/kbio.h

```diff
@@ -70,6 +70,13 @@ struct keyboard_info {
 };
 typedef struct keyboard_info keyboard_info_t;
 
+/* keyboard repeat rate mapping table */
+static const int delays[]  = { 250, 500, 750, 1000 };
```

Comment:
In other cases we have an opt-in macro guarding these so there's no pollution of the identifier space. Though even then these probably should be given less-generic names.

Reply:
Jessica, updated to avoid namespace pollution via rates and delays. Even though a certain 'pollution' via kbio.h was intended to prevent people from producing this mess again, I did not consider a more global perspective. Updated and solved.